src/api/posts/src/storage.js

```diff
@@ -0,0 +1,165 @@
+const { logger } = require('@senecacdot/satellite');
+const { redis } = require('./libs/redis');
+// Redis Keys
+const feedsKey = 't:feeds';
+const flaggedFeedsKey = 't:feeds:flagged';
+const postsKey = 't:posts';
+
+// Namespaces
+const feedNamespace = 't:feed:';
+const postNamespace = 't:post:';
+// Suffixes
+const invalidSuffix = ':invalid';
+const delayedSuffix = ':delayed';
+
+// "6Xoj0UXOW3" to "t:post:6Xoj0UXOW3"
+const createPostKey = (id) => postNamespace.concat(id);
+// "NirlSYranl" to "t:feed:NirlSYranl"
+const createFeedKey = (id) => feedNamespace.concat(id);
+// "NirlSYranl" to "t:feed:NirlSYranl:invalid"
+const createInvalidFeedKey = (id) => createFeedKey(id).concat(invalidSuffix);
+// "NirlSYranl" to "t:feed:NirlSYranl:delayed"
+const createDelayedFeedKey = (id) => createFeedKey(id).concat(delayedSuffix);
+
+module.exports = {
+  /**
+   * Feeds
+   */
+  addFeed: async (feed) => {
```

Comment:
In a follow-up PR, we can rip out the feed code from this service, and/or do it when we bring in the feed queue work (cc @c3ho).

Reply:
Sounds good